doc/release/new-policy.rst

```diff
@@ -0,0 +1,159 @@
+Release Policy Project
+======================
+
+Summary
+-------
+
+Tarantool release policy changes to become more clear and intuitive.
```

Comment:
```suggestion
The Tarantool release policy has changed to become more clear and intuitive.
```
This is clearer. The current version is harder to read ("is 'changes' a verb or a noun? hm")